components/gauge-arc/overview.md

```diff
@@ -3,145 +3,106 @@ title: Overview
 page_title: Arc Gauge Overview
 description: Overview of the Arc Gauge for Blazor.
 slug: arc-gauge-overview
-tags: telerik,blazor,chart,overview
+tags: telerik,blazor,arc,gauge,overview
 published: True
 position: 0
 ---
 
-# Chart Overview
+# Arc Gauge Overview
 
-The <a href="https://www.telerik.com/blazor-ui/chart" target="_blank">Blazor Chart component</a> allows you to visualize data to your users in a meaningful way so they can draw conclusions. You can use a variety of chart types and control all aspects of the chart's appearance - from colors and fonts, to paddings, margins and templates.
+The Telerik Arc Gauge for Blazor represents [numerical values]({%slug arc-gauge-pointers%}) on an arc [scale]({%slug arc-gauge-scale%}).
 
-#### To use a Telerik chart for Blazor, add the `TelerikChart` tag.
+This article is separated in the following sections: 
 
->caption Basic chart with series and category axis [data binding](data-bind), and a few commonly used appearance settings
+* [Basics](#basics)
 
-````CSHTML
-Basic chart and common settings/elements
+* [Features](#features)
 
-<TelerikChart>
-	<ChartSeriesItems>
-		<ChartSeries Type="ChartSeriesType.Line" Name="Product 1 (bound to simple data)" Data="@simpleData">
-		</ChartSeries>
-		<ChartSeries Type="ChartSeriesType.Line" Name="Product 2 (bound to model)" Data="@modelData" Field="@nameof(MyDataModel.SecondSeriesValue)">
-			<ChartSeriesLabels Template="#=value# in #=dataItem.ExtraData# quarter" Visible="true"></ChartSeriesLabels>
-		</ChartSeries>
-	</ChartSeriesItems>
+* [Methods](#methods)
 
-	<ChartValueAxes>
-		<ChartValueAxis Color="red"></ChartValueAxis>
-	</ChartValueAxes>
+## Basics
 
-	<ChartCategoryAxes>
-		<ChartCategoryAxis Categories="@xAxisItems"></ChartCategoryAxis>
-	</ChartCategoryAxes>
+>caption To add a Telerik Arc Gauge for Blazor to your application:
 
-	<ChartTitle Text="Quarterly sales trend"></ChartTitle>
+1. Add the `<TelerikArcGauge>` tag.
 
-	<ChartLegend Position="Telerik.Blazor.ChartLegendPosition.Bottom">
-	</ChartLegend>
-</TelerikChart>
+1. Add an instance of the `<ArcGaugePointer>` to the `<ArcGaugePointers>` collection.
 
-@code {
-	public class MyDataModel
-	{
-		public int SecondSeriesValue { get; set; }
-		public string ExtraData { get; set; }
+1. Provide a `Value` for the `<ArcGaugePointer>`.
 
-	}
+>caption Basic Telerik Arc Gauge for Blazor.
 
-	public List<MyDataModel> modelData = new List<MyDataModel>()
-    {
-		new MyDataModel() { SecondSeriesValue = 1, ExtraData = "first" },
-		new MyDataModel() { SecondSeriesValue = 5, ExtraData = "second" },
-		new MyDataModel() { SecondSeriesValue = 3, ExtraData = "third" },
-		new MyDataModel() { SecondSeriesValue = 2, ExtraData = "fourth" },
-	};
+![Basic Arc Gauge](images/basic-arc-gauge.png)
```

Comment:
I fixed the image so it does not have whitespace. Consider enabling the center template in this example, it looks cool. It can be an optional step in the list above.